pantheon-bundle/src/main/java/com/redhat/pantheon/servlet/PublishDraftVersion.java

```diff
@@ -72,12 +73,17 @@ private Locale getLocale(SlingHttpServletRequest request) {
     }
 
     private String getVariant(SlingHttpServletRequest request) {
-        return paramValue(request, "variant", DocumentVariant.DEFAULT_VARIANT_NAME);
+        return paramValue(request, "variant", null);
     }
 
     @Override
     public void run(SlingHttpServletRequest request, PostResponse response, SlingPostProcessor[] processors) {
         logger.debug("Operation Publishinging draft version started");
+        String variant = getVariant(request);
```

Comment:
@benradey  Can you please add a test case for this block in PublishDraftVersionTest.java